packages/framework/ini-files/config-specs.ini

```diff
@@ -891,6 +891,52 @@ opt-set-cmake-var Trilinos_ENABLE_ThyraEpetraAdapters BOOL FORCE : OFF
 opt-set-cmake-var Trilinos_ENABLE_ThyraEpetraExtAdapters BOOL FORCE : OFF
 opt-set-cmake-var Trilinos_ENABLE_Triutils BOOL FORCE : OFF
 
+[PACKAGE-ENABLES|COMPSIM]
```

Comment:
Does it make sense to enable a bunch of deprecated packages in a newly added configuration?

Reply:
@cgcgcg from the perspective of parity with CompSim's configuration of Trilinos, yes.